packages/peregrine/src/Router/resolveUnknownRoute.js

```diff
@@ -23,7 +21,34 @@ export default function resolveUnknownRoute(opts) {
             id: res.id,
             matched: true
         }));
-    });
+    }
+
+    if (!preloadDone) {
+        const preloaded = document.getElementById('url-resolver');
+        if (preloaded) {
```

Comment:
To be clear: if `#url-resolver` doesn't exist in the document, don't do anything?